aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -215,8 +215,9 @@ If the custom `WebHostService` code requires a service from dependency injection
 
 ## Proxy server and load balancer scenarios
 
-Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](xref:host-and-deploy/proxy-load-balancer).
+Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see <xref:host-and-deploy/proxy-load-balancer>.
```

Comment:
The leading "S" is missing on this 1st sentence:
ervices --> Services

Reply:
@scottaddie Where? I don't see it on Line 218. Another line?

Reply:
Nevermind. This must be a bug in the diff view. Here's what I see:

![image](https://user-images.githubusercontent.com/10702007/43329302-a10a9720-9185-11e8-84b6-ecff7db6f890.png)

Reply:
Interesting ... I'm good over here. Maybe that's reportable to GH, but I guess we can just keep an :eye: out for it to see if it happens a few times.

<img width="294" alt="capture" src="https://user-images.githubusercontent.com/1622880/43329421-eb826ac6-9185-11e8-94d8-3be437efa42d.PNG">